vagrant/ansible/roles/tests/tasks/log.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Log the line
```

Comment:
This is minor but instead of naming it after what it does, maybe name it after why it's doing this?

Reply:
I don't understand what you are proposing here. This is the name of the task and it normally tells what the task does. I've checked many other tasks, even on official documentation and that's the "normal" pattern. The "why" requires a bigger scope than a task and probably it's even unknown to this file. It depends on how these tasks are included from other files.

Reply:
No worries. I thought it was vague, but I guess you are saying it's supposed to be vague.

Reply:
I got confused about the "why". If you want more details about what it does, I could change it to something like "Add a log entry in site-tests.log". Please, feel free to provide any suggestion you like.